src/pages/[language]/jobs/[slug].vue

```diff
@@ -37,7 +37,7 @@
           />
           <app-button
             :label="$t('all_jobs')"
-            :to="$localeUrl({ name: 'jobs' })"
+            :to="$localeUrl({ name: 'slug', params: { slug: data.overview.page.slug } })"
```

Comment:
Because of this we decided with Frank to keep the Job Overview model to point to the Jobs landing page, that way we wouldn't have to create a block on each job to point to that page, and we didn't want to hardcode the slug here.

![image](https://github.com/voorhoede/voorhoede-website/assets/15196342/715cfe3e-4e29-4de8-8ca7-db25b3ff2755)